.github/workflows/discover-lean-pr-testing.yml

```
@@ -89,7 +92,7 @@ jobs:
             # Check if the diff contains files other than the specified ones
             if echo "$DIFF_FILES" | grep -v -e 'lake-manifest.json' -e 'lakefile.lean' -e 'lean-toolchain'; then
                 # If it does, add the branch to RELEVANT_BRANCHES
-                RELEVANT_BRANCHES="$RELEVANT_BRANCHES $BRANCH"
+                RELEVANT_BRANCHES="$RELEVANT_BRANCHES\n- $BRANCH"
```

```suggestion
                RELEVANT_BRANCHES="$RELEVANT_BRANCHES\n- $BRANCH"
```
Later on, you `echo` this: I cannot comment on the `echo`, but I think that `printf '%s\n' "$RELEVANT_BRANCHES"` has better chances of preserving line breaks.